enterprise_access/apps/subsidy_access_policy/models.py

```diff
@@ -136,24 +143,33 @@ def can_redeem(self, learner_id, content_key):
 
     def redeem(self, learner_id, content_key):
         """
+        TODO: INTEGRATION
+        We need a create_subsidy_transaction() method in the subsidy client that makes a POST
+        to the subsidy transaction creation endpoint:
+        Replace .redeem() below with create_subsidy_transaction()
+        https://enterprise-subsidy-internal.stage.edx.org/api/schema/redoc/#tag/api/operation/api_v1_transactions_create
+
         Redeem a subsidy for the given learner and content.
         Returns:
             A ledger transaction id, or None if the subsidy was not redeemed.
         """
         if self.access_method == AccessMethods.DIRECT:
             return subsidy_client.redeem(self.subsidy_uuid, learner_id, content_key)
-        if self.access_method == AccessMethods.REQUEST:
-            return subsidy_client.request_redemption(self.subsidy_uuid, learner_id, content_key)
+        # if self.access_method == AccessMethods.REQUEST:
+        #     return subsidy_client.request_redemption(self.subsidy_uuid, learner_id, content_key)
```

Comment:
Nit: how about raise NotImplementedError()?  Either that or delete the code, IMO.